deps/wazuh_testing/wazuh_testing/api.py

```diff
@@ -30,6 +31,12 @@ def callback_detect_api_start(line):
         return match.group(1)
 
 
+def callback_detect_api_start_json_format(line):
+    match = re.match(r'.*Listening on (.+)..', line)
+    if match:
+        return match.group(1)
+
+
```

Comment:
We could try to use `callback_detect_api_start`

Reply:
I deleted this callback and modified the `callback_detect_api_start` callback to match both. I tested this module and everything works OK. Later I'll execute the test suite to test if everything works as expected.